sdk/lib/atl/atldef.h

```diff
@@ -7,6 +7,15 @@
 
 #pragma once
 
+#ifdef NDEBUG
```

Comment:
NDEBUG is a define just to control the printout (to debugger) of DPRINT traces (not the DPRINT1). And "DPRINT(1)" are not used in ATL.